src/lib/containers/UserCardListRotate.tsx

```diff
@@ -16,32 +16,32 @@ export type UserCardListRotateProps = {
   token?: string
   useQueryResultUserData?: boolean
   size?: UserCardSize
-  summaryLink: string
-  summaryLinkText: string
+  summaryLink?: string
+  summaryLinkText?: string
+  selectedFacets?: FacetColumnRequest[]
 }
 
-export const getDisplayIds = (ids: string[] = [], count: number = DEFAULT_DISPLAY_COUNT) => {
+export const getDisplayIds = (ids: string[] = [], count: number = DEFAULT_DISPLAY_COUNT, storageUidKey: string) => {
```

Comment:
Ran into an issue when many UserCardListRotate components were in the portal, so now I use a separate entry for each one.